### PR TITLE
single-pool: remove minimum delegation check

### PR DESCRIPTION
**Pull request by 2501babe**

### CODE REVIEW

File: single-pool/program/tests/initialize.rs [hunk 1]
```diff
@@ -56,4 +61,56 @@ async fn fail_double_init() {
...
+    )
+    .await;
+
+    let instructions = instruction::initialize(
+        &id(),
+        &accounts.vote_account.pubkey(),
+        &context.payer.pubkey(),
+        &rent,
+        minimum_delegation - 1,
+    );
```
Comment (joncinque) [inline single-pool/program/tests/initialize.rs:100]: Maybe this is already tested elsewhere, but rather than checking against `minimum_delegation - 1`, would it be more useful to test `LAMPORTS_PER_SOL - 1` and make sure that it fails in *both* cases? Then you're sure that the pool minimum is respected

**Review by joncinque — COMMENTED** [review]
The program-level changes look good, and the tests are really great. Just one comment on a test. And also, since you're bumping up the package, can you include the Cargo.lock change?

File: single-pool/program/tests/initialize.rs (hunk 1 above)
Comment (2501babe) [reply single-pool/program/tests/initialize.rs:100]: oh `get_pool_minimum_delegation` is a helper that returns `max(stake_program_minimum_delegation, LAMPORTS_PER_SOL)`, using the constant would be clearer tho

Comment (joncinque) [reply single-pool/program/tests/initialize.rs:100]: Oh right, oops, all good then!
